templates/CRM/Member/Form/MembershipRenewal.tpl

```diff
@@ -88,7 +121,12 @@
       </tr>
       <tr class="crm-member-membershiprenew-form-block-renewal_date">
         <td class="label">{$form.renewal_date.label}</td>
-        <td>{include file="CRM/common/jcalendar.tpl" elementName=renewal_date}</td>
+        <td>{include file="CRM/common/jcalendar.tpl" elementName=renewal_date}
+        {if $priceSetId}
+        <br />
+        <span class="description hiddenElement" id="join_date_desc">{ts}If you choose a membership type for an organization for which this contact is not member of, then this date will also be used as the 'Member Since' date{/ts}</span>
```

Comment:
If you really want this description, then how about instead of 'If you choose a membership type for an organization for which this contact is not member of, then this date will also be used as the 'Member Since' date' 
we use:
"If you choose a membership type for an organization in which this contact is not member, this date will also be used as the 'Member Since' date for membership in that organization.'

Personally I think this is superfluous and potentially confusing on-screen help; it merely restates the normal use of a field as though it were somehow being treated specially. I would remove the description and the associated show/hide toggling.